data-products/src/main/scala/org/sunbird/analytics/exhaust/OnDemandExhaustJob.scala

```diff
@@ -172,6 +172,8 @@ trait OnDemandExhaustJob {
   private def zipAndEncrypt(url: String, storageConfig: StorageConfig, request: JobRequest)(implicit conf: Configuration, fc: FrameworkContext): String = {
 
     val path = Paths.get(url);
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "key" -> storageConfig.accountKey.getOrElse(""))), INFO)
+    JobLogger.log("Request is zipAndEncrypt1", Some(Map("requestId" -> request.request_id, "secret" -> storageConfig.secretKey.getOrElse(""))), INFO)
```

Comment:
@ddevadat Please avoid multiple logs. Since it is map, all the values can be put it into one log.

Reply:
removed